### PR TITLE
Use insertable streams to drop all media when in lite mode.

### PR DESCRIPTION
**Pull request by JonathanLennox**
(Rather than moving streams to inactive in SDP.)

### CODE REVIEW

File: modules/litemode/LiteModeContext.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+        const receiverStreams = receiver.createEncodedStreams();
+
+        this._setupLiteMode(receiverStreams.readable, receiverStreams.writable);
+    }
+
+    /**
+     * Setup lite mode for receiver streams
+     */
+    _setupLiteMode(readableStream, writableStream) {
```
Comment (saghul) [inline modules/litemode/LiteModeContext.js:63]: I'd inline this function, and the next even in the above one, so it does the whole setup.

**Review by saghul — APPROVED** [review]
Left a suggestion, but LGTM! Question: we want to do this for all received tracks right (audio and video) ?

File: modules/litemode/LiteModeContext.js (hunk 1 above)
Comment (JonathanLennox) [reply modules/litemode/LiteModeContext.js:63]: Ok.

**Comment by JonathanLennox** [comment]
> Left a suggestion, but LGTM! Question: we want to do this for all received tracks right (audio and video) ?

Yes, we want to turn off both audio and video decoding, which I believe is what this PR should do?

**Review by saghul — APPROVED** [review]

**Pull request merged**